AppiumForMac/AppiumForMacAppDelegate.m

```diff
@@ -31,6 +31,29 @@ @implementation AppiumForMacAppDelegate
 - (void)applicationDidFinishLaunching:(NSNotification *)aNotification
 {
     [DDLog addLogger:[DDTTYLogger sharedInstance]];
+    
+    NSFileManager *fileManager = [NSFileManager defaultManager];
```

Comment:
What is the point in having a separate log in file? The logs are anyway redirected to the terminal and can be collected with standard selenium APIs then

Reply:
I do not find the way to collect the logs from the AFM Service. I have tried with the Selenium API but it doesn't seems to be supported/implemented. I mean with the java client.

Reply:
This article describes the API for logs collection: https://appiumpro.com/editions/53 

https://github.com/appium/appium-ios-driver/tree/master/lib/device-log contains an example of logs collection for iOS.

So, how I see the right way of logs collection there:

- The server writes the stuff into stdout
- The Node mac os driver collects stdout/stderr
- The driver returns the collected logs to the client when requested

Reply:
I tried the /log API with the session id but the method is not implemented in Appium For Mac to get the logs. I just redirecting to the external file as of now so that we can have the logs into the file.
Sample Response:

{
    "status": 9,
    "sessionId": "aK6RHF7D",
    "value": {
        "message": "Could not find selector (get_log:) for path '/session/aK6RHF7D/log'."
    }
}

Reply:
of course it's not implemented (yet). But this is the right way of doing it